lib/less/functions/data-uri.js

```diff
@@ -20,6 +20,8 @@ module.exports = function(environment) {
             filePathNode.currentFileInfo.currentDirectory : filePathNode.currentFileInfo.entryPath;
 
         var fragmentStart = filePath.indexOf('#');
+        var fragment2Start = filePath.indexOf('?');
```

Comment:
Thanks for your contribution

would it be more elegant to use regular expressions here to pull out the fragment?

Also, it would be good to add tests.